rskj-core/src/test/java/co/rsk/core/BlockTest.java

```diff
@@ -72,7 +72,7 @@ void testParseRemascTransaction() {
         BlockHeader newHeader = blockFactory.getBlockHeaderBuilder()
                 .setParentHashFromKeccak256(PegTestUtils.createHash3())
                 .setEmptyUnclesHash()
-                .setCoinbase(TestUtils.randomAddress())
+                .setCoinbase(TestUtils.randomAddress("newHeaderAddress"))
```

Comment:
Absolutely minor thing, but why `newHeaderAddress`? In other similar places you used "coinbase" as value.

Reply:
actually no reason, maybe copy paste random String value as I was changing many similar calls.